3scaleAdapter/Makefile

```diff
@@ -40,3 +40,6 @@ docker-test: ## Runs the adapter
 
 docker-push: ## Push both builder and runtime image to the docker registry
 	docker push $(REGISTRY)/$(IMAGE_NAME)
+
+generate-config: ## Generates required artifacts for an out-of-process adapter based on the current .proto file
+	$(CURDIR)/scripts/generate-config.sh
```

Comment:
this will depend on the place where the `make` command is invoked - use something like this to get the directory of the Makefile: https://github.com/3scale/apisonator/blob/v2.89.0/Makefile#L1-L2

(while at it probably good to use that to refer to other files in this Makefile, such as Dockerfiles)

Reply:
Fair enough. I tend to call `make` at the root of a project always but point noted. Will fix